scripts/player_animation_controller.gd

```diff
@@ -11,9 +11,23 @@ func change_dir(dir: float):
 		animated_sprite.flip_h = true
 
 func walking(_dir: float):
```

Comment:
I would always recommend making function names verbs that describe what it is doing to avoid confusion. I would probably rename the functions either:

- play_<animation_name>_animation
- or  try_to_play_<animation_name>_animation

in order to convey you are playing (or trying to play) an animation of the given type.